src/GreenPlayer.js

```diff
@@ -27,12 +27,15 @@ class SoundPlayer extends EventEmitter {
         this.buffer = buffer;
 
         this.outputNode = null;
+        this.volumeEffect = null;
         this.target = null;
 
         this.initialized = false;
         this.isPlaying = false;
         this.isStarting = null;
         this.playbackRate = 1;
+
+        this.handleEvent = this.handleEvent.bind(this);
```

Comment:
We don't have other comments here >_<. But this could definitely use one.

Reply:
Like that handleEvent follows the event object style handler but the testing mock doesn't support object style.